tests/static_lib_depend_static_static/static_lib_depend_static_static.expect

```diff
@@ -1,6 +1,5 @@
 static0
 static1
 static1 archive xs3a
-static0 archive xs3
-static1
+static0 archive xs3a
```

Comment:
This is my mistake. We should fixe the comments in app.xc or make static0_archive() call static1()

Reply:
Fixed so that calls match the existing comments: static0 calls both static1 and static1_archive; static0_archive calls both static1 and static1_archive.